app/models/reading.rb

```diff
@@ -14,9 +14,13 @@ def self.update_or_create(work, user)
   end
 
   # called from reading controller
-  def self.mark_to_read_later(work, user)
-    reading_json = [user.id, Time.now, work.id, work.major_version, work.minor_version, true].to_json
-    REDIS_GENERAL.sadd("Reading:new", reading_json)
+  def self.mark_to_read_later(work, user, toread)
```

Comment:
Should we be calling this `mark_for_later` instead now, or would that involve changing too many things in too many places?